# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tibit/pilot-link/00-configure.in.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ur pilot-link-0.11.7/configure.in patched-pilot-link-0.11.7/configure.in
--- pilot-link-0.11.7/configure.in	2003-01-06 00:43:44.000000000 +0100
+++ patched-pilot-link-0.11.7/configure.in	2003-02-21 16:23:36.000000000 +0100
@@ -156,7 +156,7 @@
 CFLAGS="$CFLAGS $PNG_CFLAGS"
 LDFLAGS="$LDFLAGS $PNG_LIBS"
 
-AC_CHECK_LIB(png, main, [have_png=yes; PNG_LIBS="$PNG_LIBS -lpng"])
+AC_CHECK_LIB(png, main, [have_png=yes; PNG_LIBS="$PNG_LIBS -lpng -lz -lm"], [have_png=no], [-lz -lm])
 if test "$have_png" = yes; then
 	AC_CHECK_HEADER(png.h, , have_png=no)
 	if test "$have_png" = yes; then
